core/test/qa_Settings.cpp

```diff
@@ -546,8 +544,8 @@ const boost::ut::suite SettingsCtxTests = [] {
         SettingsCtx a;
         SettingsCtx b;
         expect(a == b);
-        a.time = std::chrono::system_clock::now();
-        b.time = std::chrono::system_clock::now() + std::chrono::seconds(1);
+        a.setTimePoint(std::chrono::system_clock::now());
```

Comment:
As public API, on order to be exposable via Python the `setTimePoint(..)` interface should take the canonical `uint64_t` UTC timestamp in [ns].

Reply:
This change was made to facilitate unit testing. I removed the functionality from the `SettingsCtx` context and added function for conversions instead.